packages/liveblocks-react/src/room.tsx

```diff
@@ -1036,30 +1036,6 @@ export function createRoomContext<
       const room = client.getRoom(roomId);
       if (room === null) return;
 
-      const notificationSettingsQuery = makeNotificationSettingsQueryKey(
-        room.id
-      );
-
-      if (requestsByQuery.has(notificationSettingsQuery)) {
-        requests.push(
-          room[kInternal].notifications
-            .getRoomNotificationSettings()
-            .then((settings) => {
```

Comment:
Polling room notification settings currently only serves the purpose of syncing room notification settings if a user has multiple browsers (or browser tabs) open - but syncing only happens every 5 minutes.

I have removed fetching room notification settings from the polling logic to not overcomplicate the logic for now. Keeping this logic meant making sure the error is cleared if polling returned a successful response when the fetch call from `useRoomNotificationSettings` returned an error and having to account for a few more cases than we normally would.

I am happy to introduce this back in future (with more thought on how we can make this work even more seamlessly across browser tabs) and if customers desire this feature.